// api/mockPhones.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "Коньяк КВИНТ",
    description: "Готовится из коньячных спиртов выдержанных в дубовых бочках не менее 3-х лет. Цвет светло-золотистый. В аромате хорошо выражены коньячные тона, вкус гармоничный. Состав: спирт коньячный, вода, сахарный сироп. ",
    price: 650,
    image: "/uploads/kvint3.png",
    Объём: "0.5л",
    Выдержка: "3 года",
    Крепость: "40%"
  },
  {
    id: "2",
    categoryId: "2",
    name: "Вермут Сухой",
    description: "Вино виноградное ароматизированное сухое белое. Состоит из 30 ароматических и целебных растений, среди которых полынь горькая, полынь лимонная, мята перечная, зверобой, тысячелистник, мелисса, цветы липы, ромашка, корень сельдерея, кориандр, гвоздика, ваниль. Этот напиток придется по душе почитателям MARTINI extra dry, можно потреблять как в виде коктейлей так и в чистом виде.",
    price: 750,
    image: "/uploads/buketBelii.png",
    Объём: "0.75л",
    Выдержка: "0",
    Крепость: "16%"
  },
  {
    id: "3",
    categoryId: "3",
    name: "КРИКОВА ВИНО",
    description: "Dantela - это особая серия натуральных вин, изящных и тонких, как голландское кружево. Производятся из классических сортов винограда. Sauvignon - Лёгкое, тонкое и свежее с харатерным нежным букетом с оттенками жасмина и мёда. прозрачный соломенный цвет гармонично дополняется тонким, хорошо сбалансированным вкусом",
    price: 350,
    image: "/uploads/dantSau.png",
    Объём: "0.75л",
    Выдержка: "0",
    Крепость: "40%"
  },
  {
    id: "4",
    categoryId: "4",
    name: "ШАМПАНСКОЕ",
    description: "Для производства этого игристого натурального полусладкого вина используются белый и красный виноград европейских классических сортов, выращенный в Южных и Центральных регионах Молдовы.",
    price: 400,
    image: "/uploads/crisham.png",
    Объём: "0.75л",
    Выдержка: "3 года",
    Крепость: "10%"
  },
  {
    id: "5",
    categoryId: "5",
    name: "Black Velvet Caramel ",
    description: "Главной особенностью Black Velvet Toasted Caramel является его приятный сладковатый вкус с ярко выраженными нотами карамели и ириса. Также в этом виски можно почувствовать некоторые пряности, цитрусовую цедру и нежный клиновый сироп. После глотка во рту появляется терпкий привкус, который достигается благодаря длительной выдержке в деревянных бочках.",
    price: 1400,
    image: "/uploads/velvet.png",
    Объём: "1л",
    Выдержка: "0",
    Крепость: "40%"
  },
  {
    id: "6",
    categoryId: "6",
    name: "Кошерный брэнди",
    description: "Напиток приготовлен под контролем Департамента кашрута Раввината г. Сент-Луис (США), OV, имеет статус Кошер Ле Меадрин, Ло Мевушаль, (соответствует самым строгим законам кашрута, не пастеризован). Создан по специальной технологии с использованием коньячных спиртов, выдержанных в дубовых бочках не менее 6 месяцев.",
    price: 600,
    image: "/uploads/kosher.png",
    Объём: "0.5л",
    Выдержка: "месяцев",
    Крепость: "40%"
  }
]
